syntax = "proto3";

message Number {
   float value = 1;
}

message MoreNumbers{
    float value_one = 1;
    float value_two = 2;
}

message Person{
    string name = 1;
}

message Greeting{
    string value = 1;
}

service Calculator {
    rpc SquareRoot (Number) returns (Number);
    rpc Addition (MoreNumbers) returns (Number);
    rpc Greet (Person) returns (Greeting);
}
